app/build.gradle

```diff
@@ -21,15 +23,47 @@ android {
             minifyEnabled false
             proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
         }
+        debug {
+            applicationIdSuffix ".debug"
+            buildConfigField("String", 'BANK_BASE_URL', "\"http://143.110.137.54/\"")
```

Comment:
Why do we need to do this?

Reply:
Just setting the bank api base url, it can be placed in some other place, no issues.

Reply:
Needs to update it wherever it's used accordingly to the new variable.

Reply:
I'll add it to another variable and push the changes.

Reply:
Done